definitions/variable/afolu/land-cover-change.yaml

```
@@ -0,0 +1,65 @@
+- Forest Area Change:
+    description: Annual net change in total forest area between last period and this 
+      period (difference divided by period length)
```

For consistency with other variables (see #177), I suggest to split the description and the notes
```suggestion
    description: Annual net change in total forest area
    notes: This variable should be computed as yearly average change between the previous
      and the current reported time step.
```